design/1.0.md

```diff
@@ -0,0 +1,91 @@
+I recently opened a Kubernetes Enhancement Proposal to introduce `go-flow-levee` during testing as a defense against accidental credential logging.
+See [KEP-1933](https://github.com/kubernetes/enhancements/pull/1936) details.
+Such integration demands a stable API and well-defined scope.
+
+I propose the following.
+Portions may already be implemented, but are included here as a "big picture."
+
+# 1.0 Targets
```

Comment:
Would benchmarking be a 1.0 target? I think speed should be a part of the promise we make to users. Benchmarks would let users know what to expect and would allow us to make sure that our analyzer doesn't slow down as we make changes to it.

Reply:
\shrug.

I feel until we get to more expensive approaches of cross-function / whole-program analysis, it costs what it costs.  But you're not wrong that they'd be good to have.

I'll add an `Internal Process` section and drop benchmarks in there, though.